packages/amplify_api_plugin_interface/lib/src/GraphQL/GraphQLOperation.dart

```diff
@@ -13,14 +13,13 @@
  * permissions and limitations under the License.
  */
 
-import 'package:flutter/foundation.dart';
 import '../types.dart';
 
 class GraphQLOperation<T> {
-  Function _cancel;
+  late Function _cancel;
```

Comment:
Is the usage of late correct here since it's being initialized in one and only constructor? I guess it's not really a lazy initialized var?

Reply:
Thanks great point, updated the method: 

GraphQLOperation({required this.response, required Function cancel})
      : _cancel = cancel;